API/API.py

```diff
@@ -10,7 +10,96 @@
 # this line allows python to find our module.
 sys.path.append('..\\SQL')
 from SQL.SqlQueries import *
-
+import re
+
+def parse_google_search_query(query):
```

Comment:
for this function, make sure that the positive/negative keywords being inserted into the list do not include the ++ or -- in their string.
similarly make sure that "-' is not part of the words in the excluded keywords list.
and that the quotes are now in the strings in the phrases lists.
as well as fix the small comments i added on this function